pkg/issuer/acme/dns/clouddns/clouddns.go

```diff
@@ -102,7 +102,7 @@ func NewDNSProviderServiceAccountBytes(project string, saBytes []byte) (*DNSProv
 func (c *DNSProvider) Present(domain, token, key string) error {
 	fqdn, value, ttl := util.DNS01Record(domain, key)
 
-	zone, err := c.getHostedZone(domain)
+	zone, err := c.getHostedZone(fqdn)
```

Comment:
@munnerz @kragniz I realize this was already merged, but should [ln 152](https://github.com/jetstack/cert-manager/pull/750/files#diff-566f47a7693e4c9de7cc1ae7092fa45cR152) also use `fqdn` instead of `domain`? It's not a huge blocker since domain verification will still work, but it seems removing the TXT record should behave the same as adding it.

Reply:
Yes, it should! I'll fix it in a new pr later this evening, unless someone else gets there first